src/devices/Lis3Dh/Lis3Dh.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.Gpio;
+using System.Device.I2c;
+using System.Device.Model;
+using System.Numerics;
+
+namespace Iot.Device.Lis3DhAccelerometer
+{
+    /// <summary>
+    /// LIS3DH accelerometer
+    /// </summary>
+    [Interface("LIS3DH accelerometer")]
+    public abstract class Lis3Dh : IDisposable
+    {
+        /// <summary>
+        /// Default I2C address
+        /// </summary>
+        public const int DefaultI2cAddress = 0x18;
+
```

Comment:
Don't you want to add the Secondary address as well? Even if you have the function, you may not know the pin but the address